problem_builder/templates/html/sb-review-per-question-feedback.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+
+<!-- Tips about specific questions the student got wrong. From pb-message[type=on-assessment-review-question] blocks -->
```

Comment:
This comment will be included in the resulting HTML.  You can use

    {# Your comment here. #}

in Django templates to avoid that.